Kickstarter-iOS/Views/Controllers/ManageViewPledgeViewController.swift

```diff
@@ -114,7 +109,9 @@ final class ManageViewPledgeViewController: UIViewController {
 
     self.viewModel.outputs.configurePaymentMethodView
       .observeForUI()
-      .observeValues { _ in }
+      .observeValues { [weak self] in
```

Comment:
Can we name the closure variable for better readability? To better describe `$0` in this case. Feel like we've been pretty consistent about that everywhere else.